actions/gradle-connected-integration/README.md

```diff
@@ -0,0 +1,26 @@
+# Gradle Connected Integration Action
+
+### Overview
+
+Runs Gradle connected and integration tests.
+
+### Example Usage
+
+```
+    - name: "Run Gradle connected tests in integration-2"
+        uses: broadinstitute/datarepo-actions/actions/gradle-connected-integration@0.37.0
+        env:
+          NAMESPACEINUSE: integration-2
+          TEST_TO_RUN: testConnected
+```
+
+### Environment Variables
+
+```
+NAMESPACEINUSE:
```

Comment:
Is there a reason this isn't `SCREAMING_SNAKE_CASE` like the `TEST_TO_RUN` env var?

Reply:
Yes, due to historical precedent all the actions currently expect `NAMESPACEINUSE` instead of `NAMESPACE_IN_USE` . Perhaps I could put in a follow-up PR to switch everything over at once?

Reply:
Gotcha. I'll approve and 👀 for a pr to switch everything!

Reply:
> Gotcha. I'll approve and 👀 for a pr to switch everything!

Thank you!